tensorflow-core-ops/tensorflow-core-ops.cabal

```diff
@@ -24,6 +24,10 @@ library
                 , lens-family
                 , text
   default-language:    Haskell2010
+  -- Work around https://ghc.haskell.org/trac/ghc/ticket/12175.
```

Comment:
It turns out that it's not completely fixed in ghc-8.0.2-rc1.  I filed a bug: https://ghc.haskell.org/trac/ghc/ticket/12885